common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/databinding/IsNotNullValidator.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.databinding;
+
+import org.eclipse.core.databinding.validation.IValidator;
+import org.eclipse.core.databinding.validation.ValidationStatus;
+import org.eclipse.core.runtime.IStatus;
+
+/**
+ * @author Andre Dietisheim
+ */
+public class IsNotNullValidator implements IValidator {
+
+	private IStatus invalidStatus;
+
+	public IsNotNullValidator(IStatus invalidStatus) {
```

Comment:
no docs for this istatus and also seem all other IValidators follows a different pattern.

What is the reason for the difference here ?